resources/value_sets.yml

```diff
@@ -1717,3 +1717,121 @@
+  :strength: required
+  :system: http://hl7.org/fhir/ValueSet/medicationrequest-category
+  :path: category
+- :type: CodeableConcept
+  :strength: preferred
+  :system: http://hl7.org/fhir/us/core/ValueSet/us-core-common-sdoh-assessments
+  :path: code
+- :type: CodeableConcept
+  :strength: preferred
+  :system: http://hl7.org/fhir/ValueSet/observation-codes
+  :path: code
+- :type: CodeableConcept
+  :strength: example
+  :system: http://hl7.org/fhir/ValueSet/observation-codes
+  :path: code
+- :type: CodeableConcept
+  :strength: extensible
+  :system: http://hl7.org/fhir/us/core/ValueSet/us-core-sexual-orientation
+  :path: value
+- :type: CodeableConcept
+  :strength: preferred
+  :system: http://hl7.org/fhir/us/core/ValueSet/us-core-survey-codes
+  :path: code
+- :type: CodeableConcept
+  :strength: extensible
+  :system: http://hl7.org/fhir/us/core/ValueSet/us-core-clinical-test-codes
+  :path: code
+- :type: CodeableConcept
+  :strength: extensible
+  :system: http://cts.nlm.nih.gov/fhir/ValueSet/2.16.840.1.113762.1.4.1021.32
+  :path: value
+  :extensions:
+  - http://hl7.org/fhir/us/core/StructureDefinition/us-core-genderIdentity
+- :type: Coding
+  :strength: extensible
+  :system: http://hl7.org/fhir/ValueSet/security-labels
+  :path: meta.security
+- :type: Coding
+  :strength: example
+  :system: http://hl7.org/fhir/ValueSet/common-tags
+  :path: meta.tag
+- :type: Coding
+  :strength: required
+  :system: http://hl7.org/fhir/us/core/ValueSet/us-core-tags
+  :path: meta.tag
+- :type: code
+  :strength: required
+  :system: http://hl7.org/fhir/ValueSet/questionnaire-answers-status
+  :path: status
+- :type: boolean
+  :strength: example
+  :system: http://hl7.org/fhir/ValueSet/questionnaire-answers
+  :path: item.answer.value
+- :type: CodeableConcept
+  :strength: preferred
+  :system: http://hl7.org/fhir/ValueSet/relatedperson-relationshiptype
+  :path: relationship
+- :type: CodeableConcept
+  :strength: preferred
+  :system: http://hl7.org/fhir/ValueSet/languages
+  :path: communication.language
+- :type: code
+  :strength: required
+  :system: http://hl7.org/fhir/ValueSet/request-intent
+  :path: intent
+- :type: CodeableConcept
+  :strength: example
+  :system: http://hl7.org/fhir/ValueSet/servicerequest-category
+  :path: category
+- :type: CodeableConcept
+  :strength: required
+  :system: http://hl7.org/fhir/us/core/ValueSet/us-core-servicerequest-category
+  :path: category
+- :type: CodeableConcept
+  :strength: example
+  :system: http://hl7.org/fhir/ValueSet/servicerequest-orderdetail
+  :path: orderDetail
+- :type: boolean
+  :strength: example
+  :system: http://hl7.org/fhir/ValueSet/medication-as-needed-reason
+  :path: asNeeded
+- :type: CodeableConcept
+  :strength: example
+  :system: http://hl7.org/fhir/ValueSet/participant-role
+  :path: performerType
+- :type: CodeableConcept
+  :strength: example
+  :system: http://terminology.hl7.org/ValueSet/v3-ServiceDeliveryLocationRoleType
+  :path: locationCode
```

Comment:
I assume whatever context this is being used in knows what Resource type is relevant so you don't need to technically print out the Resource type here?  Might be nice to just put it in here if you have it at generation time so this is a more useful human-readable artifact for reviews too.  Just an idea -- I might be misinterpreting what this file is for.

Reply:
This file is used to check that all of the value sets we expect to be loaded are loaded. Adding the resource type would be easy if the file weren't made by hand.

Reply:
Oh, this file is made by hand?  Why do we need `path` then?

Reply:
It just comes straight from the US Core metadata: https://github.com/inferno-framework/us-core-test-kit/blob/main/lib/us_core_test_kit/generated/v3.1.1/metadata.yml#L93